Modules/ModParser-3_0.lua

```diff
@@ -997,7 +995,7 @@ local specialModList = {
 		mod("BlockChance", "MORE", -30),
 		mod("SpellBlockChance", "MORE", -30) 
 	},
-	["maximum life becomes 1, immune to chaos damage"] = { flag("ChaosInoculation") },
+	["maximum life becomes 1, immune to chaos damage"] = {flag("ChaosInoculation") },
```

Comment:
why this ?

Reply:
Oh, woops. I was messing around with trying to get the Full Life setting to show up when you have Chaos Inoculation, just to be absolutely sure it puts you down as on Full Life. For some reason I couldn't get it to work so I just undid all the stuff I changed, but I didn't even realize I erased that space. Not a needed change, obviously!